tests/Driver/Mssql/DownCommandTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Db\Migration\Tests\Driver\Mssql;
+
+use Yiisoft\Yii\Db\Migration\Tests\Common\Command\AbstractDownCommandTest;
+use Yiisoft\Yii\Db\Migration\Tests\Support\Factory\MssqlFactory;
+
+/**
+ * @group mssql
+ */
+final class DownCommandTest extends AbstractDownCommandTest
+{
+    public function setup(): void
```

Comment:
```suggestion
    public function setUp(): void
```

And in other test classes also.